README.md

```diff
@@ -216,16 +216,45 @@ CNAB too. CNAB specifies three actions which `docker app` provides as commands:
 * `upgrade`
 * `uninstall`
 
-Here is an example installing an Application Package, querying its status and
+Here is an example installing an Application Package, inspect it and
 then uninstalling it:
 ```console
 $ docker app install examples/hello-world/example-hello-world.dockerapp --name hello
 Creating network hello_default
 Creating service hello_hello
 
-$ docker app status hello
-ID                  NAME                MODE                REPLICAS            IMAGE                        PORTS
-0m1wn7jrgkgj        hello_hello         replicated          1/1                 hashicorp/http-echo:latest   *:8080->5678/tcp
+$ docker app inspect hello
+{
+    "Metadata": {
+        "version": "0.1.0",
+        "name": "hello-world",
+        "description": "Hello, World!",
+        "maintainers": [
+            {
+                "name": "user",
+                "email": "user@email.com"
+            }
+        ]
+    },
+    "Services": [
+        {
+            "Name": "hello",
+            "Image": "hashicorp/http-echo",
+            "Replicas": 1,
+            "Ports": "8080"
```

Comment:
@silvin-lubecki is there a way to force the verbose Compose JSON format? I'm worried that if we use short or long format it will be difficult for people to consume.

Reply:
We are only using the short format but we can change it and use the long one if you like.

Reply:
We do it "manually" in the `inspect` command so yes we can choose what ever format is the most practical.